src/main/java/com/Elessar/app/util/Duration.java

```diff
@@ -0,0 +1,22 @@
+package com.Elessar.app.util;
+
+/**
+ * Created by Hans on 2/16/19.
+ */
+public class Duration {
+    private long timestamp;
+    private long duration;
```

Comment:
you probably want to use java.time.Duration here instead of a long as it contains a bunch of helper functions.